docs/jptasks.md

```diff
@@ -107,3 +107,16 @@ python main.py \
 
 * \* 1-shot setting In [PaLM 2](https://arxiv.org/abs/2305.10403)
 
+## [JAQKET v2](https://www.nlp.ecei.tohoku.ac.jp/projects/jaqket/)
```

Comment:
I don't know the background of the dataset but I am wondering if we can just name it `JAQKET` without `v2` for simplification. Or, `v2` is important? 
cc: @polm-stability

Reply:
Why I add "v2" is v1 dataset exists and it's for question answering on multiple choices:
https://sites.google.com/view/project-aio/dataset#h.9lkgxzw759e3

but we can remove "v2" until "v1" comes.

Reply:
That makes sense. Let's keep "v2" then.